connect.games.txt

```diff
@@ -1,134 +1,139 @@
-			}
-
-			"CBaseServer__RejectConnection"
-			{
-				"library"       "engine"
-				"linux"         "@_ZN11CBaseServer16RejectConnectionERK8netadr_siPKc"
-				"mac"           "@_ZN11CBaseServer16RejectConnectionERK8netadr_siPKc"
-				"windows"       "\x55\x8B\xEC\x81\xEC\x04\x05\x00\x00\x56\x6A\xFF"
-			}
-
-			"CBaseClient__SetSteamID"
-			{
-				"library"       "engine"
-				"linux"         "@_ZN11CBaseClient10SetSteamIDERK8CSteamID"
-				"mac"           "@_ZN11CBaseClient10SetSteamIDERK8CSteamID"
-				"windows"       "\x55\x8B\xEC\x8B\x55\x08\x8B\x02\x89\x41\x59\x8B\x42\x04"
-			}
-
-			"CBaseServer__CheckChallengeType"
-			{
-				"library"       "engine"
-				"linux"         "@_ZN11CBaseServer18CheckChallengeTypeEP11CBaseClientiR8netadr_siPKcii"
-				"mac"           "@_ZN11CBaseServer18CheckChallengeTypeEP11CBaseClientiR8netadr_siPKcii"
-				"windows"       "\x55\x8B\xEC\x83\xEC\x14\x56\x57\x8B\x7D\x2A\x8B\xF1"
-			}
-		}
-	}
-
-	"#default"
-	{
-		"#supported"
-		{
-			"engine"    "tf2"
-		}
-
-		"Signatures"
-		{
-			"CBaseServer__ConnectClient"
-			{
-				"library"       "engine"
-				"linux"         "@_ZN11CBaseServer13ConnectClientER8netadr_siiiiPKcS3_S3_i"
-				"mac"           "@_ZN11CBaseServer13ConnectClientER8netadr_siiiiPKcS3_S3_i"
-				"windows"       "\x55\x8B\xEC\x81\xEC\x24\x05\x00\x00"
-			}
-
-			"CBaseServer__RejectConnection"
-			{
-				"library"       "engine"
-				"linux"         "@_ZN11CBaseServer16RejectConnectionERK8netadr_siPKc"
-				"mac"           "@_ZN11CBaseServer16RejectConnectionERK8netadr_siPKc"
-				"windows"       "\x55\x8B\xEC\x81\xEC\x04\x05\x00\x00\x57"
-			}
-
-			"CBaseClient__SetSteamID"
-			{
-				"library"       "engine"
-				"linux"         "@_ZN11CBaseClient10SetSteamIDERK8CSteamID"
-				"mac"           "@_ZN11CBaseClient10SetSteamIDERK8CSteamID"
-				"windows"       "\x55\x8B\xEC\x56\x57\x8B\x7D\x08\x8B\xF1\x8D\x4E\x04"
-			}
-
-			"CBaseServer__CheckChallengeType"
-			{
-				"library"       "engine"
-				"linux"         "@_ZN11CBaseServer18CheckChallengeTypeEP11CBaseClientiR8netadr_siPKcii"
-				"mac"           "@_ZN11CBaseServer18CheckChallengeTypeEP11CBaseClientiR8netadr_siPKcii"
-				"windows"       "\x55\x8B\xEC\x83\xEC\x14\x53\x8B\x5D\x14"
-			}
-		}
-	}
-}
+"Games"
+{
+	"#default"
+	{
+		"#supported"
+		{
+			"engine"    "dods"
+			"engine"    "css"
+			"engine"    "hl2dm"
+			"engine"    "tf2"
+		}
+
+		"Offsets"
+		{
+			"ISteamGameServer__BeginAuthSession"
+			{
+				"linux"         "29"
+				"mac"           "29"
+				"windows"       "29"
+			}
+
+			"ISteamGameServer__EndAuthSession"
+			{
+				"linux"         "30"
+				"mac"           "30"
+				"windows"       "30"
+			}
+
+			"CheckMasterServerRequestRestart_Steam3ServerFuncOffset"
+			{
+				"windows"		"240"
+			}
+		}
+
+		"Signatures"
+		{
+			"CBaseServer__CheckMasterServerRequestRestart"
+			{
+				"library"       "engine"
+				"windows"       "\x55\x8B\xEC\x83\xEC\x1C\x53\x57\x33\xD2"
+			}
```

Comment:
The old one here was a midfunc sig to the right bit of the function, the new sig+offset is definitely cleaner though.

Reply:
I just realized that now 😄
Totally missed that!